src/components/Swap/Output.tsx

```diff
@@ -41,21 +41,23 @@ const Footer = styled(Row)`
   height: 1em;
 `
 
-export default function Output({ disabled, children }: PropsWithChildren<InputProps>) {
+export default function Output({ children }: { children?: ReactNode }) {
   const { i18n } = useLingui()
 
   const disableBranding = useBrandingSetting()
 
   const {
     [Field.OUTPUT]: { balance, amount: outputCurrencyAmount, usdc: outputUSDC },
+    error,
     trade: { state: tradeState },
     impact,
   } = useSwapInfo()
 
   const [swapOutputAmount, updateSwapOutputAmount] = useSwapAmount(Field.OUTPUT)
   const [swapOutputCurrency, updateSwapOutputCurrency] = useSwapCurrency(Field.OUTPUT)
 
-  const isRouteLoading = disabled || tradeState === TradeState.LOADING
+  const isDisabled = error !== undefined
+  const isRouteLoading = isDisabled || tradeState === TradeState.LOADING
```

Comment:
these few lines are duplicated between input + output above with just the Field being different